src/frontend/src/utilities/ConfirmationDialogRaw.tsx

```diff
@@ -0,0 +1,105 @@
+import ListItem from '@mui/material/ListItem';
+import ListItemText from '@mui/material/ListItemText';
+import DialogTitle from '@mui/material/DialogTitle';
+import DialogContent from '@mui/material/DialogContent';
+import DialogActions from '@mui/material/DialogActions';
+import Dialog from '@mui/material/Dialog';
+import RadioGroup from '@mui/material/RadioGroup';
+import Radio from '@mui/material/Radio';
+import FormControlLabel from '@mui/material/FormControlLabel';
+
+const options = [
+    'None',
+    'Atria',
+    'Callisto',
+    'Dione',
+    'Ganymede',
+    'Hangouts Call',
+    'Luna',
+    'Oberon',
+    'Phobos',
+    'Pyxis',
+    'Sedna',
+    'Titania',
+    'Triton',
+    'Umbriel',
+];
+
+export interface ConfirmationDialogRawProps {
+    id: string;
+    keepMounted: boolean;
+    value: string;
+    open: boolean;
+    onClose: (value?: string) => void;
+}
+
+const ConfirmationDialogRaw = (props: ConfirmationDialogRawProps) => {
+    const { onClose, value: valueProp, open, ...other } = props;
+    const [value, setValue] = React.useState(valueProp);
+    const radioGroupRef = React.useRef<HTMLElement>(null);
+
+    React.useEffect(() => {
+        if (!open) {
+            setValue(valueProp);
+        }
+    }, [valueProp, open]);
+
+    const handleEntering = () => {
+        if (radioGroupRef.current != null) {
+            radioGroupRef.current.focus();
+        }
+    };
+
+    const handleCancel = () => {
+        onClose();
+    };
+
+    const handleOk = () => {
+        onClose(value);
+    };
+
+    const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        setValue((event.target as HTMLInputElement).value);
+    };
+
+    return (
+        <Dialog
+            sx={{ '& .MuiDialog-paper': { width: '80%', maxHeight: 435 } }}
+            maxWidth="xs"
+            TransitionProps={{ onEntering: handleEntering }}
+            open={open}
+            {...other}
+        >
+            <DialogTitle>Phone Ringtone</DialogTitle>
+            <DialogContent dividers>
+                <RadioGroup
+                    ref={radioGroupRef}
+                    aria-label="ringtone"
+                    name="ringtone"
+                    value={value}
+                    onChange={handleChange}
+                >
+                    {options.map((option) => (
+                        <FormControlLabel
+                            value={option}
+                            key={option}
+                            control={<Radio />}
+                            label={option}
+                        />
+                    ))}
+                </RadioGroup>
+            </DialogContent>
+            <DialogActions>
+                <Button autoFocus onClick={handleCancel}>
+                    Cancel
+                </Button>
+                <Button onClick={handleOk}>Ok</Button>
+            </DialogActions>
+        </Dialog>
+    );
+}
+export default ConfirmationDialogRaw;
```

Comment:
How is this being used? It looks to have been copied from somewhere, as it has options that I don't think come from our design, and I see it imported into `LeafletMap.tsx` and `MapDescriptionComponents.tsx` but I don't see that import actually being used. Will it be used going forward?